include/unifex/connect_awaitable.hpp

```diff
@@ -81,7 +81,7 @@ class _sender_task<Receiver>::type {
         bool await_ready() noexcept {
           return false;
         }
-        void await_suspend(coro::coroutine_handle<promise_type>) {
+        void await_suspend(coro::coroutine_handle<promise_type>) noexcept(noexcept(((Func &&) func_)())) {
```

Comment:
I wonder if you could fix the Windows-only build error with this change.
```suggestion
        void await_suspend(coro::coroutine_handle<promise_type>) noexcept(std::is_nothrow_invocable_v<Func>) {
```